src/server/game/Entities/Unit/Unit.h

```diff
@@ -714,6 +714,14 @@ enum ReactiveType
     MAX_REACTIVE
 };
 
+struct PositionUpdateInfo
+{
+    PositionUpdateInfo() : Relocated(false), Turned(false) { }
```

Comment:
you can just add an inline initialization in both bool
`relocated = false`

Reply:
I know but I just followed the remaining struct code styles.